modin/pandas/base.py

```diff
@@ -1470,17 +1470,25 @@ def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
         Returns:
             kurtosis : Series or DataFrame (if level specified)
         """
+        axis = self._get_axis_number(axis)
         if level is not None:
-            return self._default_to_pandas(
-                "kurt",
-                axis=axis,
-                skipna=skipna,
-                level=level,
-                numeric_only=numeric_only,
-                **kwargs,
-            )
+            from .series import Series
+
+            func_kwargs = {
+                "axis": 0,
+                "skipna": skipna,
+                "level": level,
+                "numeric_only": numeric_only,
+            }
+            if isinstance(self, Series):
+                return (
+                    self.groupby(level=level, axis=axis, sort=False)
+                    .apply(func="kurt", **func_kwargs)
+                    .rename(None)
+                )
+
+            return self.apply("kurt", **func_kwargs)
```

Comment:
Is the dataframe version not a `groupby`? If it is, we can remove the check for `Series` above.

Reply:
I have found, that both Series and DataFrame objects works fine with just `apply` function, so moving code block to `series.py` is not needed now.